src/main/java/coydir/commons/core/Messages.java

```diff
@@ -6,6 +6,8 @@
 public class Messages {
     public static final String MESSAGE_UNKNOWN_COMMAND = " is an unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
+
```

Comment:
Wait why add empty line here?

Reply:
will update in the next PR with dg update